integration_tests/cleanup-jest-html-reports.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Set the number of directories to keep
+ num_of_dir_to_keep=$1
+
+ if [[ $GITHUB_WORKFLOW ]]; then
+    cd $GITHUB_WORKSPACE
+    cd jest-html-report
+    ls
+ else
+   cd ./reports/jest-html-report
+   ls
```

Comment:
Indentation seems off here. Better use 2 spaces consistently.

Reply:
Also I would leave the `cd`-ing to the calling script/user, so that this whole if-else section can probably go.

Reply:
Thanks modified the script to decide the reports path so removed the if else